libtomcrypt_VS2013.sln

```diff
@@ -0,0 +1,28 @@
+﻿
```

Comment:
We decided that we don't add VS projects for all the various versions. There's a 2008 Version and if you want to use it in a newer version, feel free to convert it yourself. _But_ who wants to use a VS project anyways when there's the msvc makefile?